### PR TITLE
N16: rib

### PR DESCRIPTION
**Pull request by bmatthieu3**
*Part of #365*

### CODE REVIEW

File: content/posts/newsletter-016/index.md [hunk 1]
```diff
@@ -67,6 +67,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [rib]
+
+![rib](rib.gif)
+
+Parsing a 3D model file and understand the different links between bone matrices,
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-016/index.md:74]: `understand` -> `understanding`?

Note that this change would take the line over 80 chars, which will make the linter unhappy, so you might need to redo the line breaks if you make this change - apologies for that!

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Looks good, just one minor grammar issue 👍

File: content/posts/newsletter-016/index.md (hunk 1 above)
Comment (bmatthieu3) [reply content/posts/newsletter-016/index.md]: of course! thank you for the typo

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:fire:

**Comment by ozkriff** [comment]
Thanks!

**Pull request merged**